packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -483,6 +429,19 @@ function ListView({
     }
   };
 
+  // Refetch the data and update the main data without loading
+  const refreshDataOnBackground = async () => {
+    setIsLoadingOnBackground(true);
```

Comment:
I think `InBackground` but what do you think of `refetchBackgroundData` and `isLoadingBackgroundData`?